cli/tests/unit/net_test.ts

```diff
@@ -43,13 +44,18 @@ Deno.test(
   },
 );
 
+function tmpUnixSocketPath(): string {
+  const folder = Deno.makeTempDirSync();
+  return join(folder, "socket");
```

Comment:
I think you either need to clean this file up manually or generate a random name like `Deno.makeTempFileSync()` does. The current CI run appears to confirm that.

Reply:
Oh, nevermind. The failure is in `http_test.ts`.

Reply:
Fixed the failure. The path is random through the use of `Deno.makeTempDirSync()`.

Reply:
Yea, I was mistaken.